pytest_flask/fixtures.py

```diff
@@ -6,8 +6,9 @@
 import socket
 
 try:
-    from urllib2 import urlopen
+    from urllib2 import URLError, urlopen
```

Comment:
A suggestion: instead of falling back on an `ImportError`, you might consider explicitly testing the python version using `sys.version_info` and use the correct imports for that version: I had problems in the past were this technique caused an import error in Python 3 for different reasons to give a confusing warning because of the fallback (pytest-dev/pytest-mock#68).

(I understand the code was already like this before but thought I would mention it anyway).